schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int NOT NULL GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(30),
+    date_of_brith date NOT NULL,
+    escape_attempts int,
+    neutered boolean,
+    weight_kg int NOT NULL
```

Comment:
- [ ] It would be better if we change the data type for `weight_kg` since SQL is strictly typed language, figures like 4.5 as weight could break it hence it should be changed to `decimal` which allows for more flexibility

Reply:
Thanks for noticing it😅